src/main/java/games/strategy/engine/lobby/server/NullModeratorController.java

```diff
@@ -97,10 +97,6 @@ public void boot(final INode node) {
     // nothing
   }
 
-  void assertUserIsAdmin() {
```

Comment:
@ron-murhammer  yes

@prastle  IP ban was discussed in: https://github.com/triplea-game/triplea/issues/1488, it's pretty easy to get around it. On the other hand mac is more difficult. Is there actually a reason to do IP banning and not mac? If we are going to use network identifiers, mac is a better one, it is sticky to a users device and is uniquely identifying. IP can be wrong and sometimes fixed even by resetting your router.

@prastle  re: password reset
First it does not make sense to have that functionality in lobby to reset passwords of players that have logged in. Beyond that, with an update of DB we need to rework it a bit. Short term a sql script can replace the functionality, so hang tight.. Medium/longer term if we share authentication with the NodeBB (new) forum, then password reset can be done via the forums (and also in which case this code would not make any sense anymore anyways and would need to be  redone in any case)